cloud/passportauth/README.md

```diff
@@ -0,0 +1,75 @@
+# RainCatcher Passport Auth
+Raincatcher authentication and authorization module using [PassportJS][1]. This module creates and initializes a Passport authentication service using Passport's local strategy.
```

Comment:
We can just inline links instead of using number references, a bit easier for maintenance. Could add a link to [local strategy](https://github.com/jaredhanson/passport-local) and decide whether to call the lib Passport or PassportJS.